### ISSUE TITLE
Prehear voices in drop-down menu

### ISSUE THREAD

**Issue by LinqLover** [LinqLover/simple-openai-tts-playground#3]
As a user of the playground, I would like to press a small speaker icon next to any voice in the drop-down menu to listen to a small constant sample of the voice, in order to compare different voices faster. These samples should be independent of the current playground configuration and only be generated once.

**Comment by Bortus-AI** [comment]
https://ttsopenai.com/ has samples for each voice so maybe integrate something like they did

![Screenshot_20240513-175210](https://github.com/LinqLover/simple-openai-tts-playground/assets/100248124/cd12f57c-909b-4090-a239-d8c719aa8cac)

**Comment by LinqLover** [comment]
Wow, I wasn't aware of that service, thanks for the pointer! That actually eliminates the need I had for this playground (except for surveying expenses by yourself through the API rather than through a third-party service). If you feel like contributing to this project, you're still welcome! I would prefer the sample UI to be as minimalistic as possible. Maybe bootstrap offers some drop-down widget where you can customize the menu items and add a speaker button next to each voice name ...

**Comment by Bortus-AI** [comment]
I agree, I prefer the sample UI myself and prefer to use my own API key and use my funds with openai than pay a 3rd party. 

Speaker button next to each voice name is a good idea.  I don't have any time currently to contribute but hopefully can in the next few weeks when my current project is finished. 

Keep up the good work

**Comment by Hassaan68** [comment]
hi @LinqLover , can you please assign this issue to me ?

**Comment by LinqLover** [comment]
Sure @Hassaan68! Mind sharing your thoughts of how you would design this feature before you go?
